source/crud/tests/v2/findOneAndDelete-hint-error.yml

```diff
@@ -0,0 +1,53 @@
+runOn:
+  # Server versions >= 4.1.10 raise errors for unknown findAndModify options (SERVER-40005)
+  - { maxServerVersion: "4.1.9" }
+
+data:
+  - { _id: 1, x: 11 }
+  - { _id: 2, x: 22 }
+
+collection_name: &collection_name 'findOneAndDelete_hint'
+
+tests:
+  -
+    description: "FindOneAndDelete with hint string unsupported (client-side error)"
+    operations:
+      -
+        object: collection
+        name: findOneAndDelete
+        arguments:
+          filter: &filter { _id: 1 }
+          hint: "_id_"
+        error: true
+    expectations:
+      -
+        command_started_event:
```

Comment:
According to the spec there should be no command started event: `For servers < 4.2, the driver MUST raise an error if the caller explicitly provides a value`

Reply:
Good catch. I'll strike these.

Reply:
FWIW, having an empty or omitted `expectations` field here won't actually test that the driver does not send a command. The [test runner implementation](https://github.com/mongodb/specifications/blob/master/source/crud/tests/README.rst) considers this field optional and only asserts as many events as are found in the array (if any). We don't have a way to assert that **no** events are emitted.

Reply:
Interesting. Although, in this case the fact that the `findOneAnd*` raised an error _does_ mean the error was client-side because if the driver sent the command the server would have ignored the "hint" option and the operation would have succeeded.